projects/addon-editor/extensions/media/utils.ts

```diff
@@ -0,0 +1,25 @@
+import {Attribute} from '@tiptap/core';
+import {Attrs, Node as NodeElement} from 'prosemirror-model';
+
+export function tuiExtractSources(node: NodeElement): Array<Array<Attrs | string>> {
+    const nodes =
+        ((node.content ?? {}) as unknown as {content: NodeElement[]})?.content ?? [];
```

Comment:
🤯

Reply:
Maybe like this, if I understood correctly?
```ts
const node: NodeElement[] = (node as any).content?.content ?? [];
```
Also why do we have to typecast here?

Reply:
fixed